example/r-template-files/README.md

```diff
@@ -0,0 +1,66 @@
+# `w-template-files`
+
+<br>
+
+While templates can be written with other code in `.ml` files, they can also
+live in their own source files.  This can be useful as templates become larger,
+or when you have many templates.
+
+If your template file is mostly HTML, you can give it a name like
+`template.eml.html`, to trigger HTML syntax highlighting by your editor.
+Additionally, if you are using `ocamlformat`, the `.html` extension will
+prevent errors that come from `ocamlformat` attempting to format the syntax of
+the template.
+
+This example does just that. It splits the code of the basic template example,
+[**7-template**](../7-template#files), into two files. The first is the
+template, in
+[`template.eml.html`](https://github.com/aantron/dream/blob/master/example/w-template-files/template.eml.html):
```

Comment:
These hyperlinks should point to `r-template-files` now, here and in all other hyperlinks.